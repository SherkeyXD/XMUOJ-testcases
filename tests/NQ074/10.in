22
15
70
9
11
65
87
87
40
70
23
26
54
21
52
70
32
18
75
91
82
64
1
