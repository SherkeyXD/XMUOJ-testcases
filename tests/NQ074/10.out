35286471
63185247
26831475
27581463
61528374
74286135
74286135
47185263
63185247
36814752
37285146
53168247
36418572
52617483
63185247
42736815
36258174
63741825
83162574
72418536
58417263
15863724
